### PR TITLE
fix(quoi-feur): check if the trigger word is formatted in md

### PR DESCRIPTION
**Pull request by dilaouid**
This member noticed that when 'quoi' is formatted in md, the bot ignores it:
image

Thought that **MAYBE** the bot should consider it. So the bot now remove all markdown.
As a result, the bot now removes all Markdown formatting. Fortunately, the md text is well-formatted, and it doesn't use unusual symbols like '<>' for emojis:

image

I also added a test to check it.

### CODE REVIEW

**Review by potb — DISMISSED** [review]

**Comment by potb** [comment]
Too many changes in here, a simple indicator is that the PR's name is too long and won't fit in a standard commit name

**Comment by dilaouid** [comment]
Fixed the stupid commit

**Comment by ayoubqrt** [comment]
@dilaouid you need to rebase and you need to run eslint autofix

File: src/modules/quoiFeur/quoiFeur.helpers.ts [hunk 1]
```diff
@@ -7,12 +7,13 @@ import {
 } from 'discord.js';
 
 import { cache } from '../../core/cache';
-import { removeEmoji, removePunctuation } from '../../helpers/regex.helper';
+import { removeEmoji, removeMarkdown,removePunctuation } from '../../helpers/regex.helper';
```
Comment (potb) [inline src/modules/quoiFeur/quoiFeur.helpers.ts:10]:
```suggestion
import { removeEmoji, removeMarkdown, removePunctuation } from '../../helpers/regex.helper';
```

**Review by potb — APPROVED** [review]

**Review by luca-montaigut — APPROVED** [review]

**Pull request merged**